profiles/models.py

```diff
@@ -1,37 +1,19 @@
-import collections
-from typing import Any, Iterable
+from flask_sqlalchemy import SQLAlchemy
 
+db = SQLAlchemy()
 
-class Client(object):
-    def __init__(self, name: str, client_id: str, client_secret: str, redirect_uri: str) -> None:
-        self.name = name
-        self.client_id = client_id
-        self.client_secret = client_secret
-        self.redirect_uri = redirect_uri
-
-    def __repr__(self) -> str:
-        return '<Client %r>' % self.name
-
+ID_LENGTH = 8
 
-class Clients(collections.Set):
-    elements = []
 
-    def __init__(self, iterable: Iterable[Client]) -> None:
-        for value in iterable:
-            if value not in self.elements:
-                self.elements.append(value)
+class Profile(db.Model):
+    id = db.Column(db.String(ID_LENGTH), primary_key=True)
+    name = db.Column(db.Text(), nullable=False)
+    orcid = db.Column(db.String(19), unique=True)
```

Comment:
from yesterday, we may want to introduce immediately a `state` field

Reply:
I did think about state and history but wanted to keep this PR as small as needed. (Since it doesn't cover deleting etc.)